style.css

```diff
@@ -0,0 +1,360 @@
+body {    
+    background-color: lightgoldenrodyellow;
+    font-size: 20px;
+    padding: 0;
+}
+
+p {
+    text-align: justify;
+}
+.contaner {
+    box-shadow: 5px 5px 5px rgba(122, 122, 122, 0.5);
+    border-radius: 20px;
+    box-sizing: border-box;
+    max-width: 1024px;
+    margin: 0 auto;
+    background-color: linen;
+}
+
+header {
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+
+    background-color: #fde4cf;
+    min-height: 400px;
+    }
+
+#foto {
+    width: 270px;
+}
+
+#contact_foto {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap-reverse;
+    padding: 10px 10px 10px 20px;
+}
+
+h1, h2 {
+    text-align: center;
+}
+
+.name {
+    font-size: 50px;
+    text-transform: uppercase;
+    font-weight: bold;
+    text-decoration: underline #66cccc;
+    word-spacing: 0.5em;
+    margin: 0 auto;
+    padding-top: 30px ;
+
+}
+
+.position {
+    background-color: lightgray;
+    max-width: 400px;
+    border-radius: 10px;
+    font-size: 30px;
+    margin: 30px auto;
+}
+
+#contacts {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.contact_link {
+    text-decoration: none;
+    color: black;
+    font-size: 1.1em;
+    }
+.mail:before {
+    content: "";
+    background-image: url(images/mail_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 20px;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.ln:before {
+    content: "";
+    background-image: url(images/linked_icon2.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 20px;
+    display: inline-block;
+    vertical-align: middle;
+    }
+
+.tel:before {
+    content: "";
+    background-image: url(images/phone_icon.png);
+    background-size: cover;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    margin-right: 20px;
+    display: inline-block;
+    vertical-align: middle;
```

Comment:
```
    content: "";
    background-size: cover;
    height: 30px;
    width: 30px;
    border-radius: 50%;
    margin-right: 20px;
    display: inline-block;
    vertical-align: middle;
```

Can it be united in one css block? To avoid duplications.
E.g. 
```
.contact_link:before {
    content: "";
    background-size: cover;
    height: 30px;
    width: 30px;
    border-radius: 50%;
    margin-right: 20px;
    display: inline-block;
    vertical-align: middle;
}
.contanct_link.gh:before {
    background-image: url(images/github_icon.png);
}
```
and same for other links with icons.

Reply:
done, thanks for advice